docs/user-guide/running.md

```diff
@@ -15,6 +15,50 @@ When your environment is running, you can [deploy your workload](deploying-workl
 
 To force device re-registration remove EdgeDevice CR from the cluster and `/var/local/yggdrasil/device` directory from the device
 
+---
+## Automatically
```

Comment:
TBH, I would move this file to here: 

https://github.com/project-flotta/project-flotta.github.io/blob/main/documentation/latest/dev/guide.md